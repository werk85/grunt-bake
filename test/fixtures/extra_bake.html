<!DOCTYPE html>
<html>
	<head></head>
	<body>
	<!--(bake-start _bake="default_bake.html > extra-page.html")-->
		<a href="{{@link}}">Open</a>
	<!--(bake-end)-->
	</body>
</html>