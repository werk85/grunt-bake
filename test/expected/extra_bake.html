<!DOCTYPE html>
<html>
	<head></head>
	<body>
		<a href="extra-page.html">Open</a>
	</body>
</html>